python/tk_multi_workfiles/entity_tree/entity_tree_form.py

```diff
@@ -78,6 +78,10 @@ def __init__(self, entity_model, search_label, allow_task_creation, extra_fields
         self._expanded_items = set()
         self._auto_expanded_root_items = set()
 
+        # load the setting that states whether the first level of the tree should be auto expanded
+        app = sgtk.platform.current_bundle()
+        self._auto_expand_tree = app.get_setting("auto_expand_tree", True)
```

Comment:
Yes! removing the default arg of True is fine, it is pulling the value from the info yml if not supplied in the env config.